fpdf/__init__.py

```diff
@@ -31,6 +31,12 @@
 from .template import Template, FlexTemplate
 from .deprecation import WarnOnDeprecatedModuleAttributes
 
+try:
+    import .ttfonts  # this module only exists in PyFPDF, it has been removed in fpdf2 since v2.5.7
+    warnings.warn("You have both PyFPDF & fpdf2 installed - To only keep fpdf2 run: pip uninstall --yes pypdf && pip install --upgrade fpdf2")
```

Comment:
I think "To only keep fpdf2 run**ning**:" is grammatically better.

Maybe we also want to explicitly say that they conflict with each other?

Reply:
Thank you for the feedback.

I improved the wording on this line